docs/modules/user-guide/partials/ref_devfile-commands.adoc

```diff
@@ -0,0 +1,108 @@
+[id="ref_devfile-commands_{context}"]
+= Devfile commands
+
+See the following tables for command properties in a devfile:
```

Comment:
```suggestion
The following tables describe command properties that you can include in a devfile:
```
Just a suggestion. Not sure if it's still accurate to what you're saying. Your call whether or not to update.